.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "rv_base"]
 	path = rv_base
-	url = ../RISCV_ISA_CoreDSL
```

Comment:
Is this necessary? `../repo` also works wih URLs, see here: https://git-scm.com/docs/git-submodule

Reply:
I don't think this is necessary, definitively not for the main repository. But I had trouble fetching the submodule in my fork, so I had to change this for myself.

Reply:
that makes sense of course, leave this in for the moment.